script/openqa-bootstrap

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -x
```

Comment:
can you make that `#!/bin/sh -e`? By default I would not make that "-x". We can leave that to the user, don't you think?

Reply:
It would give the user more info on what is happening.

Reply:
sh -ex?

Reply:
I did some tests and it seems that `sh -e` won't work because `zypper --root` installation will return `107` and also because of some potential error (that my script actually deals with) when starting the container.

Reply:
I suggest to explicitly catch these, e.g. `zypper … ||:` to explicitly ignore the exit code then

Reply:
Good point. I use `-e` in almost all my scripts and either do what oliver suggests or use the command as an if condition and actually handle the error. I would keep the `-x`, too.